test-app/tests/unit/array-test.ts

```diff
@@ -49,7 +49,7 @@ const ARRAY_SETTER_METHODS = [
 // vice versa).
 expectTypeOf<TrackedArray<unknown>>().toMatchTypeOf<Array<unknown>>();
 
-module('TrackedArray', function (hooks) {
+module.only('TrackedArray', function (hooks) {
```

Comment:
@locks assume we probably need this taken out too?

Reply:
🤦 apologies

Reply:
@locks you're fine, just figured I could flag it for you before someone who can merge comes back 😉